internal/clients/consul_cluster.go

```diff
@@ -147,7 +162,7 @@ func DeleteConsulCluster(ctx context.Context, client *Client, loc *sharedmodels.
 	return deleteResp.Payload, nil
 }
 
-// GetAvailableHCPConsulVersions gets the list of available Consul versions that HCP supports.
+// GetAvailableHCPConsulVersionsForLocation gets the list of available Consul versions that HCP supports.
 func ListConsulUpgradeVersions(ctx context.Context, client *Client, loc *sharedmodels.HashicorpCloudLocationLocation,
```

Comment:
Was this supposed to be renamed to match the doc comment?

Reply:
Yes great catch!